apps-rendering/src/components/Metadata/Metadata.defaults.tsx

```diff
@@ -0,0 +1,41 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import { remSpace } from '@guardian/source-foundations';
+import Avatar from 'components/Avatar';
+import Byline from 'components/Byline';
+import CommentCount from 'components/CommentCount';
+import Dateline from 'components/Dateline';
+import Follow from 'components/Follow';
+import { getFormat } from 'item';
+import type { Item } from 'item';
+import type { FC } from 'react';
+
+export const defaultStyles = css`
+	flex-grow: 1;
+	padding-top: ${remSpace[1]};
+`;
+
+type Props = {
+	item: Item;
+	withByline: boolean;
+	css?: SerializedStyles;
+	className?: string;
+};
```

Comment:
Are we able to use the `Styleable` helper for consistency?

```suggestion
type Props = Styleable<{
	item: Item;
	withByline: boolean;
}>;
```